docs/Reference/Pantheon-API-Methods.md

```diff
@@ -801,15 +801,15 @@ None
 
 ### eth_blockNumber
 
-Returns the index of the current block the client is processing.
+Returns the index of the block currently being processed.
```

Comment:
can you make this sentence active instead of passive to be consistent with line 812 (and guidelines)? Thanks.